projects/storefrontstyles/scss/cxbase/_variables.scss

```diff
@@ -20,6 +20,7 @@ $background: $white !default;
 $inverse: $white !default;
 $text: $black !default;
 $transparent: transparent !default;
+$max-width-container: 1140px !default;
```

Comment:
While a sass variable is already providing a lot of flexibility, I really prefer a CSS variable. We can easily do this in this line: 

```scss
$max-width-container: var(--cx-page-width-max, 1140px) !default;
```

But I'd prefer to define the css variable in `root.scss`:
```scss
:root {
  --cx-page-width-max: 1140px;
}
```

And then add the variable on all the places cross page styles.

Reply:
Done